### PR TITLE
Aggregated data popup fix

### PR DESCRIPTION
**Pull request by KipSigei**

### CODE REVIEW

File: src/connectors/ona-api/aggregateFormData.js [hunk 1]
```diff
@@ -154,6 +162,14 @@ function processFormData(formData, indicatorField, aggregateOptions) {
         prevSumTotal = previousPeriodGroupData[previousPeriodGroupData.length - 1][indicatorField]
           || 0;
         prevTotal = previousPeriodGroupData[previousPeriodGroupData.length - 1].total || 0;
+        if (extraProps && extraProps.length) {
+          extraProps.forEach((p, x) => {
+            prevExtraPropsSumTotal[x] = previousPeriodGroupData[
+              previousPeriodGroupData.length - 1][p]
+              || 0;
+            return prevExtraPropsSumTotal;
```
Comment (cKellyDesign) [inline src/connectors/ona-api/aggregateFormData.js]: @KipSigei - What is this returning to? I didn't know you could return something in a forEach

File: src/connectors/ona-api/aggregateFormData.js [hunk 2]
```diff
@@ -165,6 +181,22 @@ function processFormData(formData, indicatorField, aggregateOptions) {
         // reduce sumTotal for current groupData
         for (let x = 0; x < groupData.length; x += 1) {
           sumTotal += parseInt(groupData[x][indicatorField] || 0, 10);
+          const { parsedUID } = groupData[x];
+          parsedLocName = parsedUID;
```
Comment (cKellyDesign) [inline src/connectors/ona-api/aggregateFormData.js]: What is this being used for? Can we just pass in `parsedUID` as is into `currentPeriodaggregatedDataObj`?

File: src/connectors/ona-api/aggregateFormData.js [hunk 3]
```diff
@@ -245,6 +295,6 @@ export default function aggregateFormData(layerData, locations, filterOptions) {
   data = processFilters(layer, filterOptions);
 
   // Process data
-  aggregatedData = processFormData(data, layer.property, layer.aggregate);
+  aggregatedData = processFormData(data, layer.property, layer.aggregate, layer.extraProps);
```
Comment (cKellyDesign) [inline src/connectors/ona-api/aggregateFormData.js]: Actually, if the `extraProps` array only pertains to the data aggregation, can we nest it into `layer.aggregate` in the layer spec, keep `processFormData` to 3 parameters, and define `extraProps` once in the function?

File: src/connectors/ona-api/aggregateFormData.js (hunk 2 above)
Comment (KipSigei) [reply src/connectors/ona-api/aggregateFormData.js]: we are getting the `parsedUID` from the groupData which is not exposed when building the  `currentPeriodaggregatedDataObj` which is outside the `groupData` loop so we are assigning it to a var which is defined outside that loop.

Comment (cKellyDesign) [reply src/connectors/ona-api/aggregateFormData.js]: 👍

**Pull request merged**